SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1831,8 +1622,9 @@ void CSolver::CompleteComms(CGeometry *geometry,
               base_nodes->SetUnd_Lapl(iPoint, iVar, bufDRecv[buf_offset+iVar]);
             break;
           case SOLUTION_LIMITER:
-            for (iVar = 0; iVar < nVar; iVar++)
-              base_nodes->SetLimiter(iPoint, iVar, bufDRecv[buf_offset+iVar]);
+          case PRIMITIVE_LIMITER:
+            for (iVar = 0; iVar < COUNT_PER_POINT; iVar++)
+              (*limiter)(iPoint,iVar) = bufDRecv[buf_offset+iVar];
```

Comment:
Whats that? Is `(*limiter)` a cast, no. Or are the brackets of `(iPoint,iVar)` an access operator for the limiter container that is created a bit above (yep thats more likely)
```c++
auto limiter = &base_nodes->GetLimiter();
```

Reply:
dereferencing a pointer, I'll change it to be consistent with the other changes.